### ISSUE TITLE
Headers in lowercase

### ISSUE THREAD

**Issue by Gianfranco97** [glpi-project/glpi#3522]
* Steps to reproduce (which actions have you made) :
I have a problem trying to log in with a token. It is a bit strange, at first I thought the problem was mine but after brooding about it, I can not see what is wrong. I am trying to login with a code similar to the following:

```js
let myHeaders = new Headers()
myHeaders.append('Content-Type', 'application/json')
myHeaders.append('Authorization', 'user_token mG7y6hcMgwESTh2Z1KLRJL267bJxXsqoyXeWPbGT')

let options = {
method: 'GET',
headers: myHeaders
}
fetch ('https://dev.flyve.org/glpi/apirest.php/initSession', options)
.
.
.
```

**Request Headers:**

![captura de pantalla 2018-02-08 a la s 11 55 09 a m](https://user-images.githubusercontent.com/19965590/35982804-0c4f0154-0cc7-11e8-949e-31dfc8e2c003.png)

```
GET /glpi/apirest.php/initSession HTTP/1.1
Host: dev.flyve.org
Connection: keep-alive
Pragma: no-cache
Cache-Control: no-cache
authorization: user_token mG7y6hcMgwESTh2Z1KLRJL267bJxXsqoyXeWPbGT
Origin: http://localhost:3000
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36
content-type: application/json
Accept: */*
Referer: http://localhost:3000/app
Accept-Encoding: gzip, deflate, br
Accept-Language: en,es-419;q=0.9,es;q=0.8
DNT: 1
```

**Response Headers:**

![captura de pantalla 2018-02-08 a la s 11 55 18 a m](https://user-images.githubusercontent.com/19965590/35982821-16cb7838-0cc7-11e8-90d6-0561eb60d3dc.png)

```
HTTP/1.1 400 Bad Request
Server: nginx
Date: Thu, 08 Feb 2018 15:53:15 GMT
Content-Type: application/json; charset=UTF-8
Content-Length: 201
Connection: keep-alive
Expires: Mon, 26 Jul 1997 05:00:00 GMT
Cache-Control: no-store, no-cache, must-revalidate
Pragma: no-cache
Access-Control-Allow-Origin: *
Access-Control-Expose-Headers: content-type, content-range, accept-range
```

* Expected result :

A valid session token

* Actual result :

![captura de pantalla 2018-02-08 a la s 11 58 23 a m](https://user-images.githubusercontent.com/19965590/35982970-6783c050-0cc7-11e8-8d83-4d23f33d2111.png)
```
["ERROR_LOGIN_PARAMETERS_MISSING","parameter(s) login, password or user_token are missing; view documentation in your browser at https://dev.flyve.org/glpi/apirest.php/#ERROR_LOGIN_PARAMETERS_MISSING"]
```

-----

The problem is occurring in the header, more specifically in 'Authorization'. Glpi is not receiving it in a satisfactory way due to being in lowercase, but this is something that I am not able to control in any way since Fetch Api always passes all the headers to lowercase and there is no way to change this behavior. My question is, could exist the possibility of sending the header whether it is in lower or uppercase?

Thanks in advance.

**Comment by btry** [comment]
As far as I see, HTTP headers are case insensitive

https://stackoverflow.com/questions/5258977/are-http-headers-case-sensitive
=> https://tools.ietf.org/html/rfc7230#section-3.2

**Comment by Gianfranco97** [comment]
![captura de pantalla 2018-02-08 a la s 12 39 15 p m](https://user-images.githubusercontent.com/19965590/35985593-5d28c370-0ccd-11e8-8db6-0c90f8eddec0.png)
![captura de pantalla 2018-02-08 a la s 12 39 02 p m](https://user-images.githubusercontent.com/19965590/35985594-5d5f901c-0ccd-11e8-9412-cee26e4e61fd.png)

**Comment by ajsb85** [comment]
@DIOHz0r will propose a hotfix 👍

**Comment by trasher** [comment]
Closed in #3523
